reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -630,36 +631,43 @@ class StaticHtmlReporter : Reporter {
         val renderer = HtmlRenderer.builder().build()
         unsafe { +renderer.render(document) }
     }
-}
 
-private fun DIV.licenseLink(license: String) {
-    val licenseResourcePath = getLicenseResourcePath(license)
-    val sha1Git = HashAlgorithm.SHA1_GIT.calculate(licenseResourcePath) ?: return +license
+    private fun DIV.licenseLink(license: String) {
```

Comment:
You now need to `@Suppress("TooManyFunctions")` for the class (can be combined with the existing `@Suppress("LargeClass")` as `@Suppress("LargeClass", "TooManyFunctions")`).